src/packs/origins/Ancestries_g8kLlrl0nzciXfRF/Dragon_Knight_EbgLs9psWZ7DAzwb/Features_YvziC9ysJqSFn1bb/feature_Wyrmplate_LBaGlsH1lqNVbcC5.json

```
@@ -0,0 +1,97 @@
+{
+  "folder": "YvziC9ysJqSFn1bb",
+  "name": "Wyrmplate",
+  "type": "feature",
+  "_id": "LBaGlsH1lqNVbcC5",
+  "img": "icons/equipment/chest/breastplate-layered-gold.webp",
+  "system": {
+    "description": {
+      "value": "<p>Your hardened scales grant you damage immunity equal to your level to one of the following damage types: acid, cold, corruption, fire, lightning, or poison. You can change your damage immunity type when you finish a respite.</p>",
+      "director": "<p>The effect of this item needs to be customized to the damage type and amount of immunity desired. The attribute key to use is <code>system.damage.immunity.[damage type]</code> , where [damage type] is replaced with <span style=\"color:rgb(239, 230, 216);font-family:Signika, 'Palatino Linotype', sans-serif;font-size:14px;font-style:normal;font-variant-ligatures:normal;font-variant-caps:normal;font-weight:400;letter-spacing:normal;orphans:2;text-align:start;text-indent:0px;text-transform:none;widows:2;word-spacing:0px;-webkit-text-stroke-width:0px;white-space:normal;background-color:rgba(11, 10, 19, 0.9);text-decoration-thickness:initial;text-decoration-style:initial;text-decoration-color:initial;display:inline !important;float:none\">acid, cold, corruption, fire, lightning, or poison.</span></p>"
```

```suggestion
      "director": "<p>The effect of this item needs to be customized to the damage type and amount of immunity desired. The attribute key to use is <code>system.damage.immunity.[damage type]</code> , where [damage type] is replaced with acid, cold, corruption, fire, lightning, or poison.</p>"
```

Need to use ctrl-shift-V to paste without the formatting